lib/ConfigParseriOS.js

```diff
@@ -0,0 +1,37 @@
+
+/**
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+*/
+
+'use strict';
+
+const ConfigParser = require('cordova-common').ConfigParser;
+
+class ConfigParseriOS extends ConfigParser {
```

Comment:
```suggestion
class PlatformConfigParser extends ConfigParser {
```

IMO I think it would be better to make the class name generic. 
For example, if Android implement its own extended config parser class then it would follow identical naming conventions.

Almost as if we created and followed a platform template and would be easy to follow if we create a new platform in the future.

Reply:
Yes, I updated it